basic-with-separate-config/build.sbt

```diff
@@ -0,0 +1,29 @@
+val scalaMeterVersion = "0.7-SNAPSHOT"
+
+lazy val Benchmark = config("bench") extend Test
+
+/**  This allows running ScalaMeter benchmarks in separate sbt configuration.
+  *  It means, that when you want run your benchmarks you should type `bench:test` in sbt console.
+  */
+lazy val basic = Project(
+  "basic-with-separate-config",
+  file("."),
+  settings = Defaults.coreDefaultSettings ++ Seq(
+    name := "scalameter-examples",
+    organization := "com.storm-enroute",
+    version := scalaMeterVersion,
+    scalaVersion := "2.11.1",
+    scalacOptions ++= Seq("-deprecation", "-unchecked", "-feature", "-Xlint"),
+    publishArtifact := false,
+    libraryDependencies ++= Seq(
+      "com.storm-enroute" %% "scalameter" % scalaMeterVersion % "bench"
+    ),
+    resolvers ++= Seq(
+      "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
+      "Sonatype OSS Releases" at "https://oss.sonatype.org/content/repositories/releases"
+    ),
+    testFrameworks += new TestFramework("org.scalameter.ScalaMeterFramework"),
+    parallelExecution in Benchmark := false,
+    logBuffered := false
+  )
+) configs Benchmark settings(inConfig(Benchmark)(Defaults.testSettings): _*)
```

Comment:
To make it more explicit what's called on what, I would put `Benchmark` in the new line, wrapped with parentheses, and same with the `inConfig...` that is after `settings`:

    Project(
      ...
    ) config (
      ...
    ) settings (
      ...
    )